FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+TableView.swift

```diff
@@ -29,7 +29,7 @@ extension ComposeViewController: ASTableDelegate, ASTableDataSource {
         case (.main, .recipients(.cc)), (.main, .recipients(.bcc)):
             return !shouldShowEmailRecipientsLabel && shouldShowAllRecipientTypes ? 1 : 0
         case (.main, .password):
-            return isMessagePasswordSupported && contextToSend.hasRecipientsWithoutPubKey ? 1 : 0
+            return isMessagePasswordSupported && contextToSend.hasRecipientWithInvalidKey ? 1 : 0
```

Comment:
@sosnovsky After I changed this logic, UI test works strangely in local side(it works fine in semaphoreci side).
`setComposeSecurityMessage` function is not working properly and just append security message next to subject label. As a result, all UI tests which use `composeEmail` test fails.

Could you check in your local side and if this is also your case?

https://github.com/FlowCrypt/flowcrypt-ios/blob/c89812935ddf8fa856ef2456acde7a39b0081fe1/appium/tests/screenobjects/new-message.screen.ts#L199

Reply:
> setComposeSecurityMessage function is not working properly and just append security message next to subject label.

Same happens for me - I noticed that when all recipients are valid, then message is entered correctly. But if any recipient has invalid key and `set web portal password` cell is visible - message text is entered in subject text field.
Maybe we need some additional tap at message text view before entering message text, so cursor will be focused in the correct textfield.

Reply:
Yeah, sounds good.
Thank you for checking it